homeassistant/components/prometheus/__init__.py

```diff
@@ -355,13 +362,21 @@ def state_as_number(state: State) -> float:
             value = 0
         return value
 
-    @staticmethod
-    def _labels(state: State) -> dict[str, Any]:
-        return {
+    def _labels(self, state: State) -> dict[str, Any]:
```

Comment:
#113849 also propagated `state.object_id` as a label - do you want to do that as well?